docs/storage.md

```diff
@@ -461,40 +461,66 @@ For the full discussion about Ceph in XCP-ng, see this forum thread: <https://xc
 
 ## ISO SR
 
-You might wonder: how to upload an ISO? Unlike other solutions, you need to create a dedicated "space" for those, a specific ISO SR. To create one, it's pretty easy.
+You might wonder: how to upload an ISO? Unlike other solutions, you need to create a dedicated "space" for those, a specific ISO SR. To create a ISO SR, you have 2 possibilities:
+- Shared: A shared ISO SR is on a VM or in your dedicated storage server. It's accessible with a IP address, like 192.168.1.100
+- Local (not recommended): Local ISO SR is a directory created directly on the dom0. It's only accessible on the pool where the dom0 ith the directory on it.
 
-In [Xen Orchestra](management.md#xen-orchestra) go into "New/Storage" and select "ISO SR":
+Create a Shared ISO SR:
 
-![](https://xcp-ng.org/assets/img/screenshots/createSRISO.png)
+First, you need to create the NFS or SMB Share. The creation depends on the device you want to use to store your ISO, but here a small tutorial to mount a NFS share on Debian 11:
 
-Usually, you can use a NFS or SMB share. But if you don't have any of those, you can rely on a local ISO SR that you create yourself:
+```
+fdisk /dev/xvdb #create a new partition
+mkfs.ext4 -b 4096 /dev/xvdb1 # format the partition to ext4
+apt install nfs-kernel-server
+mkdir /ISOs
+mount /dev/xvdb1 /ISOs #mount the disks to /ISOs
+emacs /etc/fstab
```

Comment:
emacs is not the first choice for a beginner's tutorial.  (It's not personal preference: I don't use nano myself :))

```suggestion
nano /etc/fstab
```